docs/skopeo.1.md

```diff
@@ -2,13 +2,25 @@
 % Jhon Honce
 % August 2016
 # NAME
-skopeo -- Various operations with container images and container image registries
+skopeo -- Command line utility used to interact with local and remote container images and container image registries
 # SYNOPSIS
 **skopeo** [_global options_] _command_ [_command options_]
 # DESCRIPTION
-`skopeo` is a command line utility providing various operations with container images and container image registries. For example, it is able to inspect a repository on a container registry and fetch image. It fetches the repository's manifest and it is able to show you a `docker inspect`-like json output about a whole repository or a tag. This tool, in contrast to `docker inspect`, helps you gather useful information about a repository or a tag without requiring you to run `docker pull` - e.g. - which tags are available for the given repository? which labels the image has?
+`skopeo` is a command line utility providing various operations with container images and container image registries.
+
+`skopeo` can copy container images between various containers image stores, possibly converting them as necessary.  For example you can use `skopeo` to copy container images from one container registry to another.
+
+`skopeo` can convert a Docker v2.1 container image to an OCI image.
+
+`skopeo` can inspect a repository on a container registry without pulling the image. It fetches the repository's manifest and displays a `docker inspect`-like json output about a whole repository or a tag. `skopeo`, in contrast to `docker inspect`, helps you gather useful information about a repository or a tag without requiring you to run `docker pull` - e.g. - which tags are available for the given repository? Which labels the image has?
```

Comment:
I'd suggest "about a whole repository" to "about the repository".  

I think I'd cap "which tags" to "Which tags".

Change "Which labels the image has?" to "Which labels does the image have?"

Reply:
Also "without pulling the image" -> "without pulling a specific tag from the repository"

Reply:
I Fixed @TomSweeneyRedHat suggestions.
@runcom I think yours might be a little too technical.  I already changed that line with @ipbabble response `without needlessly pulling the image`.

BTW Once we have this cleaned up, I would like to write a blog using this content to reintroduce skopeo.